### DISCUSSION TITLE
Media Queries

### DISCUSSION THREAD

**Discussion by saurabh-lohiya** [fastn-stack/fastn#650] [Ideas & RFCs]
Media Queries should be added ASAP! So, then we won't need to separately code for mobile tablet device! Just adding a few lines will do

**Comment by amitu** [comment]
@saurabh-lohiya any ideas how they could work with FTD?

You can use the `classes` feature to add classes and write CSS etc. But what would FTD "native" way to support it? Any syntax suggestion?
